src/constants/index.ts

```diff
@@ -7,6 +7,7 @@ export const HEAD_CELL_MENU_WIDTH = 220;
 export const HEAD_CELL_MENU_ITEM_HEIGHT = 32;
 export const HEAD_CELL_MENU_ACTIVE_ITEM_GREEN_BORDER_HEIGHT = 16;
 export const HEAD_CELL_MENU_ITEM_ICON_WIDTH = 16;
+export const HEAD_CELL_MENU_BUTTON_CLASS = 'nebula-table-utils-head-menu-button';
```

Comment:
would be nice to agree on a common prefix for classes in this repo. I think maybe we should skip `utils`, since that is just the name of the repo. But what both sn-table and sn-pivot-table have in common is that they are nebula tables. So I suggest `nebula-table-xxx`

Reply:
I specifically used `nebula-table-utils-xxx` just because i wanted to leave a hint to consumer of that id/class that its comes from shared repo, that is why i put repo name before classes

Reply:
Yeah I guess that doesn't hurt. And if we move to a mono-repo, we could easily rename things if we want. Do you mind changing `COLUMN_ADJUSTER_CLASS` to `nebula-table-utils-column-adjuster` for consistency?

Reply:
Sure, I'll update it asap